pom.xml

```diff
@@ -6,26 +6,27 @@
 
 	<groupId>ca.corefacility.bioinformatics.irida.plugins</groupId>
 	<artifactId>example-plugin</artifactId>
-	<version>0.1.0-SNAPSHOT</version>
+	<version>0.3.0-SNAPSHOT</version>
 
 	<!-- Please fill out these properties with information about your particular plugin -->
 	<properties>
 		<!-- Information used to define properties about a plugin. Please see the PF4J docs for more details https://pf4j.org/doc/getting-started.html -->
 		<plugin.id>example-plugin</plugin.id>
 		<plugin.class>ca.corefacility.bioinformatics.irida.plugins.ExamplePlugin</plugin.class>
-		<plugin.version>0.2.0</plugin.version>
+		<plugin.version>0.3.0</plugin.version>
 		<plugin.provider>Aaron Petkau</plugin.provider>
 		<plugin.dependencies></plugin.dependencies>
-		<plugin.requires.runtime>1.0.0</plugin.requires.runtime>
+		<plugin.requires.runtime>1.1.0</plugin.requires.runtime>
 
 		<!-- The version of IRIDA your plugin must be compiled against. -->
-		<irida.version.compiletime>19.01.3</irida.version.compiletime>
+		<irida.version.compiletime>21.01-SNAPSHOT</irida.version.compiletime>
```

Comment:
I was thinking I would update it to 21.01 as once it's actually released.  Until then if someone tries to build the plugin it would fail even if they had `development` checked out and installed.

Reply:
Okay. That makes sense.